docs/scenarios_authoring/Intro.md

```diff
@@ -35,18 +35,18 @@ Configuring Nussknacker nodes is about using SpEL to a large degree; knowledge o
 
 SpEL [Spring Expression Language](https://docs.spring.io/spring-framework/docs/3.2.x/spring-framework-reference/html/expressions.html) is a powerful expression language that supports querying and manipulating data objects. What exactly does the term _expression_ mean and why is SpEL an _expression language_? In programming language terminology, an _expression_ is a union of values and functions that are joined to create a new value. SpEL only allows you to write expressions; therefore it is considered an expression language. A couple of examples:
 
-| Expression           | Result                         | Type                 |
-| ------------         | --------                       | --------             |
-| 'Hello World'        | "Hello World"                  | String               |
-| true                 | true                           | Boolean              |
-| {1,2,3,4}            | a list of integers from 1 to 4 | List[Integer]        |
-| {john:300, alex:400} | a map (name-value collection)  | Map[String, Integer] |
-| 2 > 1                | true                           | boolean              |
-| 2 > 1 ? 'a' : 'b'    | "a"                            | String               |
-| 42 + 2               | 44                             | Integer              |
-| 'AA' + 'BB'          | "AABB"                         | String               |
+| Expression           | Result                           | Type                 |
+| ------------         | --------                         | --------             |
+| 'Hello World'        | "Hello World"                    | String               |
+| true                 | true                             | Boolean              |
+| {1,2,3,4}            | a list of integers from 1 to 4   | List[Integer]        |
+| {john:300, alex:400} | a record (name-value collection) | Record[String, Integer] |
```

Comment:
In the "Type" column are shown real types presented in the Nussknacker. We don't have `Record[K, V]` type. We have:
`Record{field1: FieldType, ...}` and `Map[K, V]` so it was correct

Reply:
Changed in Type column to    Record{alex: Integer(400), john: Integer(300)}

Reply:
Hmm I see two places with this record example - one is here (type is not changed) and the second one is in `Basics` section (type is changed)

Reply:
Changed in Intro.md as well.